python/psautohint/ufoFont.py

```diff
@@ -325,8 +326,8 @@
 
 ADOBE_DOMAIN_PREFIX = "com.adobe.type"
 
-PROCESSED_LAYER_NAME = "AFDKO ProcessedGlyphs"
-PROCESSED_GLYPHS_DIRNAME = "glyphs.%s.processedGlyphs" % ADOBE_DOMAIN_PREFIX
+PROCESSED_LAYER_NAME = "%s.processedglyphs" % ADOBE_DOMAIN_PREFIX
```

Comment:
The layer name can be anything, it’s the directory name that is recommended to be lowercase and unique etc.

Reply:
I just followed https://github.com/adobe-type-tools/afdko/commit/a4aa4762bdcd4d4012226c766f6ca27ddb0b68a4, these need to be the same between afdko and psautohint.

Reply:
@anthrotype it can be anything BUT the ufonormalizer derives the **folder name** from the **layer name**, so the least disruptive solution is to adjust the name of the layer.

Reply:
i think it does that only when the directory name contains a non-lowercase character, but anyway as you prefer.